community/python2-tkinter/APKBUILD

```diff
@@ -13,7 +13,7 @@ arch="all"
 license="custom"
 subpackages="$pkgname-tests python2-idle:_idle:noarch"
 depends=""
-makedepends="expat-dev libressl-dev zlib-dev ncurses-dev bzip2-dev
+makedepends="expat-dev openssl-dev zlib-dev ncurses-dev bzip2-dev
```

Comment:
is openssl-dev really needed?

Reply:
The dependencies are the same as in `python2`.